contrib/julia-config.jl

```diff
@@ -23,6 +23,8 @@ function libDir()
     end
 end
 
+private_libDir() = joinpath(libDir(), "julia")
```

Comment:
Ok, then I think we should just go with what I have here and improve this script in the future as needed.